fb_clone/spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# require 'rails_helper'
+
+# RSpec.describe CommentsController, type: :controller do
+#   describe 'GET #update' do
+#     it 'returns http success' do
+#       get :update
+#       expect(response).to have_http_status(:success)
+#     end
+#   end
+
+#   describe 'GET #destroy' do
+#     it 'returns http success' do
+#       get :destroy
+#       expect(response).to have_http_status(:success)
+#     end
+#   end
+
+#   describe 'GET #new' do
+#     it 'returns http success' do
+#       get :new
+#       expect(response).to have_http_status(:success)
+#     end
+#   end
+
+#   describe 'GET #create' do
+#     it 'returns http success' do
+#       get :create
+#       expect(response).to have_http_status(:success)
```

Comment:
Commented out code should be removed from the project.
It’s better to delete commented code. If you need to review a previous version of your code, use git history
Please remove before merging and do same for all code that applies (not rails generated comments. But code you yourselves have commented out)